objectivec/src/ort_value.mm

```diff
@@ -5,7 +5,7 @@
 
 #include <optional>
 
-#include "safeint/SafeInt.hpp"
+#include "SafeInt.hpp"
```

Comment:
this probably requires updating some other places too. e.g., here:
https://github.com/microsoft/onnxruntime/blob/b1e1b25e04b0ae5f5e0af003933850e6c2122d8c/tools/ci_build/github/apple/objectivec/assemble_objc_pod_package.py#L30

Reply:
I don't know how to deal with it. If I updated objectivec/src/ort_value.mm without changing assemble_objc_pod_package.py, then the ios packaging pipeline would fail.
So I chose to not delete cmake/external/SafeInt, and reverted this change. But,  then, the ios static analysis pipeline starts to fail. :-( I don't know what I can do.

Reply:
trying to fix it here:
https://github.com/microsoft/onnxruntime/pull/13698